products/quarkus.md

```diff
@@ -26,8 +26,8 @@ releases:
 -   releaseCycle: "2.16"
```

Comment:
RHBQ has now a end date for 2.13.x (Next release of Red Hat build of Quarkus or Dec 14, 2025), see https://access.redhat.com/support/policy/updates/jboss_notes#p_quarkus.